plugins/CoreHome/javascripts/dataTable_rowactions.js

```diff
@@ -178,7 +178,7 @@ DataTable_RowAction.prototype.trigger = function (tr, e, subTableLabel, original
     }
 
     // ascend in action reports
-    if (subtable.closest('div.dataTableActions').length) {
```

Comment:
Were we just missing the `dataTableActions` class (e.g. it being removed in past) and it would work with it or we still have the class but need to be more specific with the selector?

Reply:
I actually wasn't pretty sure about it. Maybe a bit of both. It's not the normal actions datatable, but a goal visualization with actions, so it doesn't have the `dataTableActions`, but the goal viz related classes. Goal metrics for actions have been added somewhen end of last year, so guess this was kind of missed and the row evolution didn't ever work there.